### PR TITLE
feat(privateca): generate library

### LINKED ISSUE #8026: Certificate Authority API: `privateca.googleapis.com`

### PR DESCRIPTION
**Pull request by scotthart**
fixes #8026

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/8028)

### CODE REVIEW

File: google/cloud/privateca/README.md [hunk 1]
```diff
@@ -38,22 +41,24 @@ this library.
...
-  auto client = privateca::Client(privateca::MakeConnection(/* EDIT HERE */));
+  auto client = privateca::CertificateAuthorityServiceClient(
+      privateca::MakeCertificateAuthorityServiceConnection());
 
-  auto const project = google::cloud::Project(argv[1]);
-  for (auto r : client.List /*EDIT HERE*/ (project.FullName())) {
+  auto const ca_pool = "projects/" + std::string(argv[1]) + "/locations/" +
+                       std::string(argv[2]) + "/caPools/" +
+                       std::string(argv[3]);
+  for (auto r : client.ListCertificates(ca_pool)) {
```
Comment (coryan) [inline google/cloud/privateca/README.md:61]: Seems like `ListCaPools` would have been easier to implement and to understand:

https://cloud.google.com/certificate-authority-service/docs/reference/rpc/google.cloud.security.privateca.v1#google.cloud.security.privateca.v1.ListCaPoolsRequest

File: google/cloud/privateca/doc/main.dox [hunk 2]
```diff
@@ -2,8 +2,11 @@
 
 @mainpage Certificate Authority API C++ Client Library
 
-An idiomatic C++ client library for the [Certificate Authority API][cloud-service-docs], a service
-to The Certificate Authority Service API is a highly-available, scalable service that enables you to simplify and automate the management of private certificate authorities (CAs) while staying in control of your private keys.
+An idiomatic C++ client library for the
+[Certificate Authority API][cloud-service-docs], a service
```
Comment (coryan) [inline google/cloud/privateca/doc/main.dox:6]: This does not read right.  Maybe:

```
... [Certificate Authority API][cloud-service-docs].
Automate the management of private certificate authorities while staying in control of your private keys.
```

Maybe the same change in the README file too?

**Review by coryan — CHANGES_REQUESTED** [review]
The main.dox change is needed, the other stuff is your call.

**Review by scotthart — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/8028)* status: 0 of 40 files reviewed, 2 unresolved discussions (waiting on @coryan)

___
*[google/cloud/privateca/README.md, line 61 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/8028#-MtoA6aVbsDhI48xvwAG-r1-61:-MtoPAr37frBRfVlvbeR:b-896fix) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/e883f873bf6314370f7a02937c6fb79b1cc6c346/google/cloud/privateca/README.md#L61)):*
<details><summary><i>Previously, coryan (Carlos O'Ryan) wrote…</i></summary><blockquote>

Seems like `ListCaPools` would have been easier to implement and to understand:

https://cloud.google.com/certificate-authority-service/docs/reference/rpc/google.cloud.security.privateca.v1#google.cloud.security.privateca.v1.ListCaPoolsRequest
</blockquote></details>

Done.
___
*[google/cloud/privateca/doc/main.dox, line 6 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/8028#-MtoA6aVbsDhI48xvwAQ-r1-6:-MtoOzxW7CZv4XoNHUXv:b-wngw5o) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/e883f873bf6314370f7a02937c6fb79b1cc6c346/google/cloud/privateca/doc/main.dox#L6)):*
<details><summary><i>Previously, coryan (Carlos O'Ryan) wrote…</i></summary><blockquote>

This does not read right.  Maybe:

```
... [Certificate Authority API][cloud-service-docs].
Automate the management of private certificate authorities while staying in control of your private keys.
```

Maybe the same change in the README file too?
</blockquote></details>

README file was ok; missed this one.

**Review by coryan — APPROVED** [review]
> Reviewed 37 of 40 files at r1, 3 of 3 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/8028)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @scotthart)

**Pull request merged**